gateway/conf.d/apicast.conf

```diff
@@ -82,23 +82,23 @@ location @upstream {
 }
 
 location / {
-  set $cached_key null;
-  set $credentials null;
-  set $usage null;
-  set $service_id null;
-  set $proxy_pass null;
-  set $secret_token null;
+  set $cached_key '';
```

Comment:
There's something I don't understand about this. I see in the commit comment that you're doing this to avoid sending this in headers for example. Why are they sent when set to `null` and not sent when set to `''` ?

Reply:
Nginx is not going to send values that are empty strings. But it was sending `null`. Which was pretty ugly. Imo better to keep it empty than send `null` as a string.

Reply:
:+1: 
That's a surprising behavior.